robottelo/vm.py

```diff
@@ -244,7 +244,7 @@ def enable_repo(self, repo, force=False):
         """
         downstream_repo = None
         if repo in (REPOS['rhst6']['id'], REPOS['rhst7']['id']):
-            downstream_repo = settings.sattools_repo
+            downstream_repo = settings.sattools_repo[self.distro[:5]]
```

Comment:
I am wondering if we should pick rhel6 when `rhst6` repo is the repo id (same for rhel7) instead of basing the decision on the distro.